cli/dcoscli/subcommand.py

```diff
@@ -10,25 +12,9 @@ def _default_modules():
     :rtype: {}
     """
 
-    # avoid circular imports
-    from dcoscli.auth import main as auth_main
-    from dcoscli.config import main as config_main
-    from dcoscli.help import main as help_main
-    from dcoscli.marathon import main as marathon_main
-    from dcoscli.node import main as node_main
-    from dcoscli.package import main as package_main
-    from dcoscli.service import main as service_main
-    from dcoscli.task import main as task_main
-
-    return {'auth': auth_main,
-            'config': config_main,
-            'help': help_main,
-            'marathon': marathon_main,
-            'node': node_main,
-            'package': package_main,
-            'service': service_main,
-            'task': task_main
-            }
+    defaults = subcommand.default_subcommands()
+    return {s: __import__('dcoscli.{}'.format(s), fromlist=['main']).main
+            for s in defaults}
```

Comment:
I am not super exited with respect to the `__import__`. Let's use `importlib.import_module`: https://docs.python.org/3.5/library/importlib.html#importlib.import_module